apps/src/templates/teacherNavigation/TeacherNavigationPaths.tsx

```diff
@@ -25,45 +30,84 @@ export const LABELED_TEACHER_NAVIGATION_PATHS = {
     url: TEACHER_NAVIGATION_PATHS.progress,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.progress),
     label: i18n.progress(),
+    icon: 'chart-line',
   },
   textResponses: {
     url: TEACHER_NAVIGATION_PATHS.textResponses,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.textResponses),
     label: i18n.teacherTabStatsTextResponses(),
+    icon: 'pen-line',
   },
   assessments: {
     url: TEACHER_NAVIGATION_PATHS.assessments,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.assessments),
     label: i18n.assessments(),
+    icon: 'star',
   },
   projects: {
     url: TEACHER_NAVIGATION_PATHS.projects,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.projects),
     label: i18n.studentProjects(),
+    icon: 'code',
   },
   stats: {
     url: TEACHER_NAVIGATION_PATHS.stats,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.stats),
     label: i18n.teacherTabStats(),
+    icon: 'chart-simple',
   },
   manageStudents: {
     url: TEACHER_NAVIGATION_PATHS.manageStudents,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.manageStudents),
     label: i18n.roster(),
+    icon: 'users',
   },
   loginInfo: {
     url: TEACHER_NAVIGATION_PATHS.loginInfo,
     absoluteUrl: getAbsolutePath(TEACHER_NAVIGATION_PATHS.loginInfo),
     label: null,
+    icon: 'users',
```

Comment:
In this and the next two paths, there is no real "icon" provided because it is not listed in the figma.  If I left this blank, I got an error.  Right now `users` is a placeholder icon.